### DISCUSSION TITLE
How to delete a resource from the Web UI desktop?

### DISCUSSION THREAD

**Discussion by gramunno** [gravitational/teleport#47331] [Q&A] [answered]
Once a resource is present on the Web UI desktop, how can I delete It?
Thanks

**Comment by stevenGravy** [comment, marked as answer]
That is not currently supported. You can remove resources that have not expired or are dynamic with `tctl rm <type>/<name/uuid>` from CLI.

**Reply by gramunno** [reply]
Thanks!
